src/components/CreateProject/ProjectForm/CreateProjectModal.cy.tsx

```diff
@@ -1,13 +1,11 @@
 import { messages } from '@camp/messages';
 import { noop } from '@fullstacksjs/toolbox';
 
+import { createProjectFormIds } from './CreateProjectForm';
 import { CreateProjectModal } from './CreateProjectModal';
 
 const requiredFieldMessage = messages.projects.validation.required;
```

Comment:
This way is better.
```suggestion
const { minLength: minLengthMessage, required: requiredFieldMessage } =
  messages.families.validation;
```

Reply:
Me neither

Reply:
ok guys it seems I did it somehow xD
it used to be this
```ts
const requiredFieldMessage = 'این فیلد ضروری است';
const minLengthMessage = 'نام خانواده باید حداقل ۳ حرف باشد';
```
and I just ran a find and replace in vscode and it turned out this way:
```ts
const requiredFieldMessage = messages.families.validation.required;
const minLengthMessage = messages.families.validation.minLength;
```

Reply:
What's the problem with this?